### ISSUE TITLE
Enhance MiKo_3105 codefix to detect specific scenarios

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#485]
MiKo_3105 codefix provider should be aware of following situation:
```c#
var result = ObjectUnderTest.DoSomething();

Assert.AreEqual(xyz, result.Xyz);
```
as well as:
```c#
Assert.AreEqual(xyz,  ObjectUnderTest.DoSomething());
```

It should detect an access to a variable that gets assigned by the return value of a call to the object under test and fix it correctly:
```c#
var result = ObjectUnderTest.DoSomething();

Assert.That(result.Xyz, Is.EqualTo(xyz));
```
resp.
```c#
Assert.That(ObjectUnderTest.DoSomething(), Is.EqualTo(xyz));
```

**Comment by RalfKoban** [comment]
hint:
Variable: VariableDeclarator -> EqualsValueClause -> InvocationExpression -> SimpleMemberAccessExpression
Assert: SimpleMemberAccessExpression

**Comment by RalfKoban** [comment]
Also detect if a variable contains the term "expected" and consider that one as the expected value.
